### PR TITLE
Feat: Portal Management API

### PR DESCRIPTION
**Pull request by acgoldsmith**
### Description

Customers are able to integrate portal management operations with their automation systems (such as CI/CD pipelines) by utilizing a refactored and newly-published AIP-compliant Portal Management API.

### Testing instructions

Preview link: 
*  new spec URL /api/portal-managment/latest
* https://deploy-preview-6388--kongdocs.netlify.app/konnect/api/portal-management/latest/ (NB: spec will not render but demonstrates new URL)
* https://deploy-preview-6388--kongdocs.netlify.app/api/ > new card for spec available + Insomnia button 
* https://deploy-preview-6388--kongdocs.netlify.app/konnect/api/ > new Portal Management API in nav w/ link to spec

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/_data/docs_nav_konnect.yml [hunk 1]
```diff
@@ -339,6 +339,13 @@
           absolute_url: true
           generate: false
           target_blank: true
+    - text: Portal Management API
+      items:
+      - text: API Spec
+        url: /konnect/api/portal-management/latest/
+        absolute_url: true
+        generate: false
+        target_blank: true
```
Comment (cloudjumpercat) [inline app/_data/docs_nav_konnect.yml:348]: These rows need another indent

**Review by cloudjumpercat — CHANGES_REQUESTED** [review]
@acgoldsmith Just had one small comments about indentation, everything else looks good!

**Review by cloudjumpercat — APPROVED** [review]
LGTM!

**Comment by acgoldsmith** [comment]
@c3pko , this is ready to release. Please ensure the API product and spec are published from devportal: https://cloud.konghq.com/us/api-products/f99f0f29-be4e-4fc8-8cec-64e5d3511bff/versions/43fca68c-f83d-4cf1-8c40-87574d423313

**Comment by acgoldsmith** [comment]
note to self: in order to trigger the sync from /portal-api/ repo to populate the insomnia button, please merge https://github.com/Kong/docs.konghq.com/pull/6359 first

**Comment by acgoldsmith** [comment]
@rosskukulinski , I've added the insomnia button as well and confirmed it's working. API Product and version at https://cloud.konghq.com/us/api-products/f99f0f29-be4e-4fc8-8cec-64e5d3511bff/versions/43fca68c-f83d-4cf1-8c40-87574d423313 need to be published ahead of this pr.

**Pull request merged**